index.html

```diff
@@ -44,8 +46,216 @@ <h2 class="headline-subprimary">I'm Software Developer</h2>
+        </div>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="about-myself-details">
+        <h1>About Me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <button class="btn">Get my resume</button>
+      </div>
+      <svg
+        width="327"
+        height="2"
+        viewBox="0 0 327 2"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path d="M0 1H327" stroke="white" />
+      </svg>
+      <div class="about-myself-tools">
+        <ul class="tools">
+          <li>
+            <h2>
+              <svg
+                width="27"
+                height="26"
+                viewBox="0 0 27 26"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect
+                  x="0.772461"
+                  y="12.7279"
+                  width="18"
+                  height="18"
+                  transform="rotate(-45 0.772461 12.7279)"
+                  fill="#FF6B00"
+                />
+              </svg>
+              Language
+            </h2>
+            <ul class="subcategories">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </li>
+          <li>
+            <h2>
+              <svg
+                width="18"
+                height="19"
+                viewBox="0 0 18 19"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect y="0.455811" width="18" height="18" fill="#FF6B00" />
+              </svg>
+              Framework
+            </h2>
+            <ul class="subcategories">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+            </ul>
+          </li>
+          <li>
+            <h2>
+              <svg
+                width="18"
+                height="19"
+                viewBox="0 0 18 19"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect
+                  y="0.455811"
+                  width="18"
+                  height="18"
+                  rx="9"
+                  fill="#FF6B00"
+                />
+              </svg>
+              Skills
+            </h2>
+            <ul class="subcategories">
+              <li>Database Managament</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>API Design</li>
+              <li>Web Development</li>
+            </ul>
+          </li>
+        </ul>
       </div>
     </section>
```

Comment:
Fix the Icons and the alignment of the title and rest of the page 
![Screenshot from 2023-07-11 12-02-20](https://github.com/memicelis/my-portfolio/assets/118063058/872dadf1-d1aa-4998-8105-c489e13daf20)
![Screenshot from 2023-07-11 12-02-28](https://github.com/memicelis/my-portfolio/assets/118063058/4b1c708b-c951-49bc-ac50-00a6968c647b)